inventory/aix_lvgs_inventory

```diff
@@ -0,0 +1,117 @@
+#usmvg:
+#LV NAME             TYPE       LPs     PPs     PVs  LV STATE      MOUNT POINT
+#usrLocal            jfs2       64      128     2    closed/syncd    /usr/local
+#usrLclItslb         jfs2       8       16      2    closed/syncd    /usr/local/itslb
+#
+
+import re
+
+
+def parse_aix_lvgs(info):
+
+    header_active, header_details = u"[Volume_Groups_Active]", u"[Volume_Groups_Details]"
+    if info[0][0] != header_active:
+        raise Exception('Expecting a (possibly empty) list of active volume groups starting with ' +
+                        header_active)
+    info = info[1:]
+    delimiter_indices = [index for index, line in enumerate(info) if line[0] == header_details]
+    if len(delimiter_indices) > 1:
+        raise Exception('We do not expect to have more than one section starting with ' +
+                        header_details)
+
+    if len(delimiter_indices) == 1:
+        active_volume_groups = [line[0] for line in info[:delimiter_indices[0]]]
+        details = info[delimiter_indices[0] + 1:]
+    else:
+        active_volume_groups = []
+        details = info
+
+    lsvg_conf = {}
+    for line in details:
+        real_line = line[0]
+        if (real_line.endswith(":")):
+            vgname = real_line[:-1]
+            lsvg_conf.update({vgname: {}})
+            if vgname in active_volume_groups:
+                lv_state = "active"
+            else:
+                lv_state = "inactive"
+        # Some versions send a title line "LV NAME  ..."
+        elif real_line.startswith("LV") and real_line.endswith("MOUNT POINT"):
+            continue
+        else:
+            indices = [0, 20, 31, 39, 47, 52, 66, None]
+            raw_columns = [real_line[indices[i - 1]:indices[i]] for i in range(1, len(indices))]
+            lv, lvtype, num_lp, num_pp, num_pv, act_state, mountpoint = [
+                column.strip() for column in raw_columns
+            ]
+
+            # split lv state into two relevant values
+            activation, mirror = act_state.split("/")
+            if mountpoint == "N/A":
+                mountpoint = None
+            if lvtype == "N/A":
+                lvtype = None
+            lsvg_conf[vgname].update({
+                lv: (lvtype, int(num_lp), int(num_pp), int(num_pv), activation, mirror, mountpoint,
+                     lv_state)
+            })
+    return lsvg_conf
+
+
+def sort_by_logical_volume_name(volumes):
+    convert = lambda text: int(text) if text.isdigit() else text.lower()
+    alphanum_key = lambda volume: [
+        convert(c) for c in re.split('([0-9]+)', volume['logical_volume_name'])
+    ]
+    return sorted(volumes, key=alphanum_key)
+
+
+def inv_aix_lvgs(info, inventory_tree):
+    unsorted_volumes = []
+    node = inv_tree_list(".hardware.storage.logical_volumes:")
+    result = parse_aix_lvgs(info)
+    for volume_group_name, volumes in result.iteritems():
+        for logical_volume_name in volumes:
+            keys = [
```

Comment:
You can make this a global constant, and choose an immutable data type:
_KEYS = ("logical_volume_name", .... )

But it seems to me this whole block should be incorporated in the parse function.